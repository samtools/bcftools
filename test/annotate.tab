3	3212016	CTT	C,CT	indel_3212016	.	.	.	1
4	3258448	TACACACAC	T	indel_3258448	.	.	.	1
4	4000000	T	C	id1	.	.	.	1
4	4000001	T	C,A	id2	.	.	.	1
2	3199812	G	GTT,GT	indel_3199812	.	.	.	1
1	3000150	C	CT	indel_3000150	.	.	.	1
1	3000150	C	T	snp_3000150	999	1,2	1e-10,2e-10	.
1	3000151	C	T	snp_3000151	999	1	2e-10	.
1	3062915	G	T,C	snp_3062915	999	1	2e-10	.
1	3062915	GTTT	G	indel_3062915	.	.	.	1
1	3106154	A	C	snp_3106154	999	1	2e-10	.
1	3106154	C	CT	indel_3106154	.	.	.	1
1	3106154	CAAA	C	indel_3106154	.	.	.	1
1	3157410	GA	G	indel_3157410	.	.	.	1
1	3162006	GAA	G	indel_3162006	.	.	.	1
1	3177144	G	.	ref_3177144	999	1	2e-10	.
1	3177144	G	T	snp_3177144	999	1	2e-10	0
1	3184885	TAAAA	TA,T	indel_3184885	.	.	.	1
