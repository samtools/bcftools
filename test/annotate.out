##fileformat=VCFv4.1
##FILTER=<ID=PASS,Description="All filters passed">
##INFO=<ID=TEST,Number=1,Type=Integer,Description="Testing Tag">
##FORMAT=<ID=TT,Number=A,Type=Integer,Description="Testing Tag, with commas and \"escapes\" and escaped escapes combined with \\\"quotes\\\\\"">
##INFO=<ID=DP4,Number=4,Type=Integer,Description="# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases">
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
##FORMAT=<ID=GQ,Number=1,Type=Integer,Description="Genotype Quality">
##FORMAT=<ID=DP,Number=1,Type=Integer,Description="Read Depth">
##FORMAT=<ID=GL,Number=G,Type=Float,Description="Genotype Likelihood">
##FILTER=<ID=q10,Description="Quality below 10">
##FILTER=<ID=test,Description="Testing filter">
##contig=<ID=1,assembly=b37,length=249250621>
##contig=<ID=2,assembly=b37,length=249250621>
##contig=<ID=3,assembly=b37,length=198022430>
##contig=<ID=4,assembly=b37,length=191154276>
##test=<ID=4,IE=5>
##reference=file:///lustre/scratch105/projects/g1k/ref/main_project/human_g1k_v37.fasta
##readme=AAAAAA
##readme=BBBBBB
##INFO=<ID=AC,Number=A,Type=Integer,Description="Allele count in genotypes">
##INFO=<ID=AN,Number=1,Type=Integer,Description="Total number of alleles in called genotypes">
##INFO=<ID=INDEL,Number=0,Type=Flag,Description="Indicates that the variant is an INDEL.">
##INFO=<ID=STR,Number=1,Type=String,Description="Test string type">
##INFO=<ID=T_STR,Number=1,Type=String,Description="Test String">
##INFO=<ID=T_INT,Number=.,Type=Integer,Description="Test Integer">
##INFO=<ID=T_FLOAT,Number=.,Type=Float,Description="Test Float">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	A	B
1	3000150	snp_3000150	C	T	999	PASS	AN=4;AC=2;T_INT=1,2;T_FLOAT=1e-10,2e-10	GT:GQ	0/1:245	0/1:245
1	3000151	snp_3000151	C	T	999	PASS	AN=4;AC=2;T_INT=1;T_FLOAT=2e-10	GT:DP:GQ	0/1:32:245	0/1:32:245
1	3062915	indel_3062915	GTTT	G	12.9	q10	DP4=1,2,3,4;AN=4;AC=2;INDEL;STR=test	GT:GQ:DP:GL	0/1:409:35:-20,-5,-20	0/1:409:35:-20,-5,-20
1	3062915	snp_3062915	G	T,C	999	test	TEST=5;DP4=1,2,3,4;AN=3;AC=1,1;T_INT=1;T_FLOAT=2e-10	GT:TT:GQ:DP:GL	0/1:0,1:409:35:-20,-5,-20,-20,-5,-20	2:0,1:409:35:-20,-5,-20
1	3106154	indel_3106154	CAAA	C	342	PASS	AN=4;AC=2;INDEL	GT:GQ:DP	0/1:245:32	0/1:245:32
1	3106154	indel_3106154	C	CT	59.2	PASS	AN=4;AC=2;INDEL	GT:GQ:DP	0/1:245:32	0/1:245:32
1	3157410	indel_3157410	GA	G	90.6	q10	AN=4;AC=4;INDEL	GT:GQ:DP	1/1:21:21	1/1:21:21
1	3162006	indel_3162006	GAA	G	60.2	PASS	AN=4;AC=2;INDEL	GT:GQ:DP	0/1:212:22	0/1:212:22
1	3177144	snp_3177144	G	T	999	PASS	AN=4;AC=2;T_INT=1;T_FLOAT=2e-10	GT:GQ:DP	0/0:150:30	1/1:150:30
1	3177144	ref_3177144	G	.	999	PASS	AN=4;AC=0;T_INT=1;T_FLOAT=2e-10	GT:GQ:DP	0/0:150:30	0/0:150:30
1	3184885	indel_3184885	TAAAA	TA,T	61.5	PASS	AN=4;AC=2,2;INDEL	GT:GQ:DP	1/2:12:10	1/2:12:10
2	3199812	indel_3199812	G	GTT,GT	82.7	PASS	AN=4;AC=2,2;INDEL	GT:GQ:DP	1/2:322:26	1/2:322:26
3	3212016	indel_3212016	CTT	C,CT	79	PASS	AN=4;AC=2,2;INDEL	GT:GQ:DP	1/2:91:26	1/2:91:26
4	3258448	indel_3258448	TACACACAC	T	59.9	PASS	AN=4;AC=2;INDEL	GT:GQ:DP	0/1:325:31	0/1:325:31
4	4000000	id1	T	A,C	59.9	PASS	AN=4;AC=2;INDEL	GT:GQ:DP	0/1:325:31	0/1:325:31
4	4000001	id2	T	A	59.9	PASS	AN=4;AC=2;INDEL	GT:GQ:DP	0/1:325:31	0/1:325:31
